Library/ShoKanri.Http/Dtos/Transaction/TransactionCreateDto.cs

```diff
@@ -0,0 +1,8 @@
+namespace ShoKanri.Http.Dtos.Transaction;
+
+public class TransactionCreateDto
+{
+    public decimal Amount { get; set; }
+    public string Description { get; set; }
+    public int TransactionType { get; set; } // 0=Income, 1=Expense, 2=Transfer
```

Comment:
```suggestion
    public TransactionType Type { get; set; } // 0=Income, 1=Expense, 2=Transfer
```
Utiliza o enum que você criou, gostei da ideia.